src/mscorlib/src/System/Runtime/InteropServices/GcHandle.cs

```diff
@@ -7,8 +7,14 @@ namespace System.Runtime.InteropServices
     using System;
     using System.Runtime.CompilerServices;
     using System.Threading;
-    using System.Runtime.Versioning;
     using System.Diagnostics.Contracts;
+#if BIT64
+    using nuint = System.UInt64;
+    using nint = System.Int64;
+#else
+    using nuint = System.UInt32;
```

Comment:
Can we use signed nint consistently everywhere - since the handle is signed IntPtr?